content/spin/v2/python-components.md

```diff
@@ -307,24 +299,23 @@ command = "spin py2wasm app -o app.wasm"
 If you are still following along, please go ahead and update your `app.py` file one more time, as follows:
 
 ```python
-from spin_http import Response
-from spin_redis import redis_del, redis_get, redis_incr, redis_set, redis_sadd, redis_srem, redis_smembers
-from spin_config import config_get
-
-
-def handle_request(request):
-
-    redis_address = config_get("redis_address")  # fetches from `variables`
-    redis_set(redis_address, "foo", b"bar")
-    value = redis_get(redis_address, "foo")
-    redis_del(redis_address, ["testIncr"])
-    redis_incr(redis_address, "testIncr")
-
-    redis_sadd(redis_address, "testSets", ["hello", "world"])
-    content = redis_smembers(redis_address, "testSets")
-    redis_srem(redis_address, "testSets", ["hello"])
-
-    assert value == b"bar", f"expected \"bar\", got \"{str(value, 'utf-8')}\""
+from spin_sdk.http import simple
+from spin_sdk.http.simple import Request, Response
+from spin_sdk import redis
+
+class IncomingHandler(simple.IncomingHandler):
+    def handle_request(self, request: Request) -> Response:
+        with redis.open(config_get("redis_address").decode) as db:
```

Comment:
Is `config_get` correct?  Asking because (1) we seem to be veering a bit inconsistenly between `thing.f()` and `thing_f()` across ourAPIs and (2) v2 WITs changed this name to `variables`.